internal/core/data/device_test.go

```diff
@@ -169,19 +166,16 @@ func newMockDb() interfaces.DBClient {
 		return start == 0xBADF00D
 	}), mock.Anything, mock.Anything).Return(nil, fmt.Errorf("some error"))
 
-
```

Comment:
I added the spacing to try and distinguish between different method mocks since the code is extremely samey. Do we want any sort of separator so it's not just uniform single line gaps or is this fine?

Reply:
IMHO a single line of spacing is fine.